src/librustc_passes/diagnostics.rs

```diff
@@ -223,12 +223,43 @@ To fix this, add a label specifying which loop is being broken out of:
 ```
 'foo: while break 'foo {}
 ```
+"##,
+
+E0571: r##"
+The `break` statement can take an argument (which will be the value of the loop
```

Comment:
A long error explanation is supposed to start by a short summary of the error. The error explanation comes just after the erroneous code example and just before the fix.